components/SmoothLoadingImage.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  
+    <SmoothLoadingImageFadeTransition><div v-show="!isVisible"><SVGLoadingIcon /></div></SmoothLoadingImageFadeTransition>
+   <SmoothLoadingImageFadeTransition><img v-show="isVisible" :data-testid=testId :class="`object-${objectStyling} rounded-${borderRadius} w-${width} h-${height}`" :src="src" @load="showImg()"/></SmoothLoadingImageFadeTransition>
```

Comment:
Linting Police 🚓

- This lines have weird indentation (maybe it's only on Github but make sure the indentation is correct)
- On line for the line is to long and the attributes should be under each other for better readability like this:
```
    <SVGHamburgerMenuIcon
      data-testid="hamburger-menu-icon"
      alt="hamburger menu"
      class="h-8 w-8"
      @click="openMenu()"
    />
```  

Other example: 
```
    <button
      data-testid="hamburger-menu-close-button"
      class="px-2 py-.5 mr-2"
      @click="closeMenu()"
    >
      <svg
        class="stroke-primary"
        width="20"
        heigh="20"
        viewBox="4 0 15 25"
      >
        <path
          stroke-width="3"
          fill="none"
          d="M6.25,6.25,17.75,17.75"
        />
        <path
          stroke-width="3"
          fill="none"
          d="M6.25,17.75,17.75,6.25"
        />
      </svg>
    </button>
```